embulk-jruby-strptime/src/main/java/org/embulk/spi/time/RubyDateParse.java

```diff
@@ -0,0 +1,283 @@
+package org.embulk.spi.time;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RubyDateParse
```

Comment:
Thank you for suggesting. I renamed this class with `TimeZoneConverter`. I also add comments about Github URL links to easy to catch up. bf1100b1d9c95265bcbfb261abfa4e4b11c08c73 and d8c4956cbd189e2fed2632eb403f0dee37a9c014